ftests/explorer-ftests-nomode/src/test/java/org/nuxeo/functionaltests/explorer/nomode/ITExplorerTest.java

```diff
@@ -406,4 +409,25 @@ public void testInvalidArtifactPages() {
         openAndCheck(getArtifactURL(OperationInfo.TYPE_NAME, "foo"), true);
     }
 
+    /**
+     * @since 20.3.0
+     */
+    @Test
+    public void testSitemap() throws IOException {
+        driver.get(NUXEO_URL + ExplorerHomePage.URL + Distribution.SITEMAP);
+        try {
+            String ref = AbstractExplorerTest.getReferenceContent(Paths.get("data/sitemap.xml"));
+            // replace dates
+            String today = new SimpleDateFormat("yyyy-MM-dd").format(new Date());
```

Comment:
More modern: `LocalDate.now().format(DateTimeFormatter.ISO_LOCAL_DATE)`.
But you probably want a UTC date instead of a local one.